docs/istio_support.md

```diff
@@ -0,0 +1,37 @@
+# AKO on Istio
```

Comment:
please mention that this feature is in tech preview in this release

Reply:
added tech preview to title